pkgs/tools/nix/nixci/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv
+, rustPlatform
+, fetchCrate
+, callPackage
+, libiconv
+, openssl
+, pkg-config
+, Security
+, devour-flake
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nixci";
+  version = "0.1.3";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-sM/1G1mf+msWbG4CX/pZNt4FmSKR2hWXdcq5h7W1AM0=";
+  };
+
+  cargoSha256 = "sha256-PKBNQKuWV4PE7iSKr+LugayroFjDBT4/vyyjJiw/E+I=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libiconv openssl ]
+    ++ lib.optional stdenv.isDarwin Security;
```

Comment:
```suggestion
    ++ lib.optionals stdenv.isDarwin [ Security ];
```

Reply:
Will do. Just waiting for @ofborg to start and finish [the queued up task](https://github.com/NixOS/nixpkgs/pull/244487/checks?check_run_id=15269845221) (if I push a new commit, I believe it will cancel and re-queue it, causing further delays).

Reply:
Don't or it will take ages. There's an issue with Darwin builds and the waiting time is extremely long.